### ISSUE TITLE
Feature Request: Combine "Control System Wizard" into the main Advanced Track page

### ISSUE THREAD

**Issue by wjw42** [oli414/AdvancedTrack#17]
While what is currently in place is completely functional, having to create each control system, move to the right of the screen to choose the action type, and then back to the left is not my favorite. I think those options could be added to the bottom of the main Advanced Track page.

![advanced track page example](https://user-images.githubusercontent.com/81606842/214402091-eda75f79-80f0-493e-9bc7-1fc40ba13b94.png)

Here's a rough mockup I made. Here, if _Lift/Drop Track_ were selected then the bottom area would be blank (or the boxes greyed out or something).

**Comment by oli414** [comment]
Yeah I noticed the placement of the pop-up window not to be optimal. I'll see what I can do, I might just make sure that the sub window spawns within the advanced track window.
